tests/test_manylinux.py

```diff
@@ -146,21 +161,15 @@ def test_build_repair_numpy(docker_container):
     docker_exec(manylinux_id, repair_command)
     filenames = os.listdir(io_folder)
 
-    if policy == 'manylinux1':
-        expected_files = 2
-    elif policy == 'manylinux2010':
-        expected_files = 3  # We end up repairing the wheel twice to manylinux1
-
-    assert len(filenames) == expected_files
-    # Regardless of build environment, wheel only needs manylinux1 symbols
```

Comment:
I was gonna ask a question about this behaviour change but it's addressed in the commit message of 46619cd0df32f9548cbdb6e1b8d6f6f9de6d07c5. Thanks for catching that.